Bottom: 65782966b35feb375cbc386a006fbd0bddeef738
Top:    ccc67a2f1d6fa0fbafcd95a860dfd5c8d94e1fb8
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2014-07-05 17:15:24 +0100

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
